sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,20 +1,98 @@
 
 ### Features Added
 
+#### Plugin API
+
+Identity v2 provides a top-level `useIdentityPlugin` method, which allows using two new plugin packages:
+
+- [@azure/identity-vscode](https://www.npmjs.com/package/@azure/identity-vscode), which provides the dependencies of `VisualStudioCodeCredential` and enables it.
+- [@azure/identity-cache-persistence](https://www.npmjs.com/package/@azure/identity-cache-persistence), which provides persistent token caching.
+- If the `@azure/identity-vscode` plugin isn't used through the `useIdentityPlugin` function, the `VisualStudioCodeCredential` exposed by Identity v2 will throw a `CredentialUnavailableError`.
+
+Most credentials on Identity v2 now include a constructor option called `tokenCachePersistenceOptions`. If set to `tokenCachePersistenceOptions: { enabled: true }`, it will enable persistent token caching on that credential instance.
+
+#### New credentials
+
+Identity v2 includes three new credential types:
+
+- `AzurePowerShellCredential`, which re-uses any account previously authenticated with the `Az.Account` PowerShell module.
+- `ApplicationCredential`, which is a simplified `DefaultAzureCredential` that only includes `EnvironmentCredential` and `ManagedIdentityCredential`.
+- `OnBehalfOfCredential`, which enables the [On-Behalf-Of authentication flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow).
+
+#### Changes to all credentials
+
+Identity v2 enables:
+
+- Support for claims challenges resulting from [Continuous Access Enforcement (CAE)](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-continuous-access-evaluation) and [Conditional Access authentication context](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/granular-conditional-access-for-sensitive-data-and-actions/ba-p/1751775). To disable this behavior, set the environment variable `AZURE_IDENTITY_DISABLE_CP1` to any value. For more about claims challenges, see [Claims challenges, claims requests, and client capabilities](https://docs.microsoft.com/azure/active-directory/develop/claims-challenge).
+- Multi-tenant authentication on all credentials except `ManagedIdentityCredential`. To disable it, set the environment variable `AZURE_IDENTITY_DISABLE_MULTITENANTAUTH`. For more about multitenancy, see [Identity management in multitenant apps](https://docs.microsoft.com/azure/architecture/multitenant-identity/).
+
+#### Changes to interactive credentials
+
+Changes have been made to `InteractiveBrowserCredential` and `DeviceCodeCredential`:
+
+- Interactive authentication can be controlled by setting `disableAutomaticAuthentication` to `true` on the constructor of these credentials. This will prevent `getToken` from prompting for user interaction.
+- These credentials now have a new `authentication()` method, which always triggers user interaction.
```

Comment:
Should we re-use the content in https://github.com/Azure/azure-sdk-for-js/blob/04e92bc9ca9e3a0d57f816bb2a788ad2d9c53156/sdk/identity/identity/CHANGELOG.md#new-features-1 on this subject? That was very clear on what this setting does where as now, it is not that clear

Reply:
Can do! I’m worried about the length of this, but I understand your point.